docs/devguide/create-module.asciidoc

```diff
@@ -77,6 +77,66 @@ the metricsets. This file does not have `_test.go` in the name because otherwise
 
 To see an example of the `testing.go` file, look at the https://github.com/elastic/beats/tree/master/metricbeat/module/mysql[mysql module].
 
+[float]
+===== Test a Metricbeat module manually
+
+To test a Metricbeat module manually, follow the steps below.
+
+First we have to build the docker image which is available for the modules. As an example let's take MySQL module.
+
+This steps assume you have checked out the Beats repository from Github and are inside `beats` directory:
+
+```
+cd metricbeat/module/mysql/_meta/
+docker build .
+```
+
+This changes your directory to where the Dockerfile is for MySQL and builds the docker image. In the end you should see something like:
```

Comment:
Ah, I can see that you add it here. It's common to _give context and then execute_. Readers usually expect that paragraphs over/before code introduce them on what they are going to write while paragraphs after code should follow a line that was open before the code or just introduce the next piece.

```
If you see this:                <- No context, line open.
code
Then you                        <- The context is here, but you warned the reader before
```